program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+- Drawbacks are that it is out of date (based on earlier release of GCC), doesn't meet coding standards, and lacks testing
+
+
+
+### Standard RISC-V tool chain
+This is the standard RISC-V GCC tool chain whose attributes include:
+- The development is hosted by the FSF (Free software foundation). It is completely up to date with GCC and binutils. There are many commits per day.
+- It has no Pulp features
+- A precompiled version of that toolchain is available from Embecosm and possibly others. From Embecosm, the latest version is updated weekly. The latest table release is 10.2.0
+- It is currently used in the Ashling IDE product.
+
+### OpenHW CORE-V Reference Tool Chain
+This is a separate project in OpenHW managed within the SW TG. 
+- The project starts from the standard GCC tools (FSF) and adds CORE-V instructions
+- The binutils-gdb repository in OpenHW github currently has 2 branches
+ - Stable (currently, this repository is a mirror of FSF).
+ - Development (first commits to this repository will have support for CORE-V hardware loop)
+
+
+
+
+## List of project outputs
+
+*Need a list of outputs, i.e.*
+- *Documents to be produced*
+- *Downloadable code to be produced*
+
+
+## TGs Impacted/Resource requirements
+
+This work will be planned and discussed in the SW TG. The project plan will be produced by the project manager with assistance of SW TG, and reviewed/approved in TWG
+
+
+## OpenHW engineering staff resource plan: requirement and availability
+
+- Florian Zaruba - technical contribution and project planning
+- Duncan Bees - program management oversight
+
+## Engineering resource supplied by members - requirement and availability
+
+*The intent here is to write down specific time/person requirements. This is not yet available, and will be supplied in the project plan. As of Preliminary Project Proposal, the major gap identified is a resource for an initial application to test the IDE.*
+
+## OpenHW marketing resource - requirement and availability
+
+- Initial artwork suppplied by Publitek
+- No additional resource requirements yet identified
+
+## Marketing resource supplied by members - requirement and availability
+- No  resource requirements yet identified
+
+## Funding supplied by OpenHW - requirement and availability 
+- No funding requirements yet identified
+
+
+## Funding supplied by members - requirement and availability
+- No funding requirements yet identified
+
+
+
+## Architecture diagram
+*Need diagram showing SW architecture and also real-world physical setup for intended use. Action to Alexander* 
+
+## Who would make use of OpenHW output
+CORE-V CDT will be used by developers created code for the following target platforms
+- CV32E40P FPGA platform
+- *expand/detail this list*
+
+## Project license model
+
+This project will be released under EPL 2.0
+* Explain how the code under other licenses, such as GNU PL, will be incorporated *

```

Comment:
This is well understood - almost all Eclipse versions use either GNU or Clang/LLVM. This comment opens a can of worms that is not needed. Just put "the tools will be licensed under their existing licenses, as is standard for Eclipse based IDEs"